packages/sitecore-jss-react/src/components/FieldMetadata.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
+
+/** The field metadata */
+export interface FieldMetadata {
+  contextItem?: FieldMetadataContextItem | null | undefined;
```

Comment:
I see that all the properties have a type: <type> | null | undefined.
Did you ask somebody about type of values that can be passed? Initially, I thought that if metadata is provided, so all the values should be present, not really sure if they are optional. These properties are required for Pages